lend/src/components/Referral/index.tsx

```diff
@@ -56,12 +56,15 @@ const CopyToClipboardButton = styled(Button)`
 const TwitterButton = styled(Button)`
   background: #1da1f229;
   color: ${colors.buttons.secondaryText};
-  width: 24%;
+  width: 20.75%;
   border: none;
   border-radius: 0;
-  padding: 20px;
+  padding-top: 18px;
```

Comment:
can combine paddings here

Reply:
Fixed in [dedc273](https://github.com/ribbon-finance/ribbon-frontend/pull/624/commits/dedc2738370b87aa5d70a8894eb84e998b852743)